src/layout/routes.js

```diff
@@ -0,0 +1,43 @@
+import { isAuthorized } from "../router/guards/index.js";
```

Comment:
**Issue**: The 'isAuthorized' function is imported and used directly in route guards. This creates a direct dependency on the specific implementation of authorization logic. <br> **Fix**: Use dependency injection for the authorization function to make the routing configuration more flexible and decoupled from specific implementation details. <br> **Code Suggestion**: 
 ```
 - import { isAuthorized } from "../router/guards/index.js";
 + constructor({ authorizationGuard }) {
 + this.isAuthorized = authorizationGuard;
 ```